### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by izgzhen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,6 +106,14 @@ are still up-to-date.
 
 To build a GHC source distribution tarball, run Hadrian with the `sdist-ghc` target.
 
+#### Installation
+
+To install GHC artifacts, first build everything, then run the `install` target.
```
Comment (snowleopard) [inline README.md]: Why is the first step needed? All required artefacts should be (re)built automatically if need be, before copying, via Shake's tracking. Users shouldn't have to remember these things.

File: README.md [hunk 2]
```diff
@@ -122,12 +130,11 @@ zero (see [#197][test-issue]).
 Current limitations
 -------------------
 The new build system still lacks many important features:
-* There is currently no support for the `dynamic` build way: [#4][dynamic-issue].
```
Comment (snowleopard) [inline README.md]: There is currently no support for dynamic linking on Windows.

File: README.md (hunk 1 above)
Comment (izgzhen) [inline README.md]: That's a great point -- but this is only partially supported now. I should fix it later

Comment (izgzhen) [inline README.md]: Fixing should be simple, I will do that first.

Comment (snowleopard) [inline README.md]: Thanks! Ping me when ready.

**Comment by izgzhen** [comment]
This one is also updated.

**Comment by snowleopard** [comment]
Thanks, merged!

**Pull request merged**